postcss.config.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  plugins: [require('autoprefixer')]
+};
```

Comment:
please rename this to postcss.config.js and update the contents:

```suggestion
import autoprefixer from 'autoprefixer';

export default {
  plugins: [autoprefixer()]
};
```

Reply:
Done, didn't know it supported ESM!